src/SPC/builder/unix/library/imap.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\unix\library;
+
+use SPC\exception\FileSystemException;
+use SPC\exception\RuntimeException;
+use SPC\exception\WrongUsageException;
+use SPC\store\FileSystem;
+use function Safe\xdiff_file_patch_binary;
+
+trait imap
+{
+    /**
+     * @throws FileSystemException
+     * @throws RuntimeException
+     * @throws WrongUsageException
+     */
+    protected function build(): void
+    {
+        //        ext-imap is NOT thread safe, do not build with ZTS builds
+        //        if ($this->builder->getOption('enable-zts')) {
+        //            return;
```

Comment:
I would add this `if` but throw a runtime exception with the error like in the comment instead.

Reply:
made it a wrong usage extension, as the user should make sure to pass compatible options